apps/devchoices-next/pages/question/[slug].tsx

```diff
@@ -82,12 +89,25 @@ export function QuestionPage(props: QuestionPageProps) {
 
   const onLeft = () => {
```

Comment:
You can regroup both function into one 

```suggestion
  const onClick = (position) => {
    const form = new FormData()
    form.append('position', position)
    fetch(`http://localhost:8000/?slug=${slug}`, {
      method: 'POST',
      body: form,
    })
    const leftValues = position === '0' ? voteValues[0] + 1 : voteValues[0]
    const rightValues = position === '1' ? voteValues[1] + 1 : voteValues[1]
    setVoteValues([leftValues, rightValues])
    setShowResult(true)
    }
```